pkgs/development/tools/build-managers/bazel/bazel_3/default.nix

```diff
@@ -22,6 +22,8 @@
 , file
 , substituteAll
 , writeTextFile
+# for execlog binary
+, openjdk
```

Comment:
no need to introduce a third JDK, we already have `buildJdk` and `runJdk`, just use the latter.

Reply:
done